web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -40,6 +40,9 @@
 </template>
 
 <script>
+import { loggedIn } from '@/rest';
+import girderRest from '@/rest';
```

Comment:
Something the linter caught:

```suggestion
import girderRest, { loggedIn } from '@/rest';
```